server/core/src/main/java/dev/slimevr/tracking/processor/skeleton/Localizer.kt

```diff
@@ -414,9 +415,7 @@ class Localizer(humanSkeleton: HumanSkeleton) {
 	// update the hmd position and rotation
 	private fun updateSkeletonPos(travel: Vector3) {
 		var rot = Quaternion.IDENTITY
-		if (skeleton.headTracker != null) {
-			rot = skeleton.headTracker.getRotation()
-		}
+		skeleton.headTracker?.let { rot = it.getRotation() }
```

Comment:
This can be done with an elvis operator. I would suggest it but you didn't modify the last line lol, look at my last comment for reference.